src/plugins/kernel_install/utils.sh

```diff
@@ -197,7 +197,7 @@ function collect_deploy_info()
   bootloader="[bootloader]=$bootloader"
 
   # Get distro
-  distro=$(cat /etc/*-release | grep -w 'ID\(_LIKE\)\?' | cut -d = -f 2 | xargs echo)
+  distro=$(grep < /etc/*-release --word-regexp 'ID\(_LIKE\)\?' | cut --delimiter = --fields 2 | xargs printf)
```

Comment:
Using `< /etc/*-release` results in the following error:

```bash
bash: /etc/*-release: ambiguous redirect
```

Try copying the command inside `$()` to see the error.

Te reason is that `<` works differently from `cat`, as `<` is a redirect that assumes a single file source, while `cat` accepts multiple file sources. That being said, I don't know if there is a way to use `<` along with the wildcard `*` , so maybe we will have to stick with `cat`...